flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -1028,6 +1061,119 @@ class Table(
   }
 }
 
+class FlatAggTable(
```

Comment:
I hesitate to remove `FlagAggTable`. 
- Even `FlatAggTable` and `GroupedFlatAggTable` share same methods now, I'm not sure if there is a chance that `FlatAggTable` will be different from `GroupedFlatAggTable` later. At that time, we should add `FlatAggTable` back which will break compatibility.
What do you think?

Reply:
Can you describe when FlatAggTable may be needed? I'm fine to keep it if you're definitely sure it will be added back in the future. Otherwise I would suggest to remove it and only add it when necessary.

Reply:
Ok, we can remove `FlatAggTable ` if we tend to only support select in `FlatAggTable ` and `GroupedFlatAggTable `.